app/Labels/LabelServiceProvider.php

```diff
@@ -157,15 +158,18 @@ private function setUpReadModels(Container $container): void
             function () use ($container): ReadRepositoryInterface {
                 $labels = file_exists(__DIR__ . '/../../config.excluded_labels.php') ? require __DIR__ . '/../../config.excluded_labels.php' : [];
 
-                return new GodUserReadRepositoryDecorator(
-                    new JsonReadRepository(
-                        $container->get('dbal_connection'),
-                        new StringLiteral(self::JSON_TABLE),
-                        new StringLiteral(self::LABEL_ROLES_TABLE),
-                        new StringLiteral(UserPermissionsServiceProvider::USER_ROLES_TABLE),
-                        new InMemoryExcludedLabelsRepository($labels ?? [])
+                return new AppConfigReadRepositoryDecorator(
```

Comment:
Maybe make this name also more explicit as is done for the other repositories and make it clear this is specific for labels?

Reply:
Simon and I just followed the naming pattern of every specific repository interface and its existing implementations. The one for Labels never contain the word "Labels", just like almost every other class/interface in that namespace. I feel like it would be better to keep it consistent for now, and add the name "Label" at a later point to all classes/interfaces inside that namespace?